### DISCUSSION TITLE
Updating our contribution guidelines

### DISCUSSION THREAD

**Discussion by mpguerra** [ZcashFoundation/zebra#8344] [Brainstorming]
We have a number of both implicit and explicit best practices for developing on zebra dotted around in various places. 

The ECC also has some good documents for contributors to their projects. We should collect and consolidate all of these into some document that we can link to from CONTRIBUTING.md to help set expectations for external contributors. 

Let's use this discussion to collect these best practices and discuss which we might want to include in a guide for external contributors.

**Comment by mpguerra** [comment]
Examples of PR templates we might want to take inspiration from

**Reply by mpguerra** [reply]
https://github.com/zcash/lightwalletd/blob/master/.github/pull_request_template.md

**Comment by mpguerra** [comment]
Examples of CONTRIBUTING.md files

**Reply by mpguerra** [reply]
https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk/blob/main/CONTRIBUTING.md

**Reply by mpguerra** [reply]
https://github.com/zcash/zcash/blob/master/CONTRIBUTING.md (points to [Development Guidelines — Zcash Documentation 5.8.0 documentation](https://zcash.readthedocs.io/en/latest/rtd_pages/development_guidelines.html))

**Reply by arya2** [reply]
https://github.com/tokio-rs/tokio/blob/master/CONTRIBUTING.md

**Comment by mpguerra** [comment]
I have this on my TODO list
